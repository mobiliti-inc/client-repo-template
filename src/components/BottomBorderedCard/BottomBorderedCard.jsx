import styles from './BottomBorderedCard.scss';

export const BottomBorderedCard = ({ leftText, rightText, }) => (
	<div styleName="content">
		<div styleName="content">
			{ leftText }
		</div>
		<div styleName="content-right">
			{rightText}
		</div>
	</div>
);

BottomBorderedCard.propTypes = {
	leftText: PropTypes.oneOfType([PropTypes.node, PropTypes.string]).isRequired,
	rightText: PropTypes.string.isRequired,
};
export default CSSModules(BottomBorderedCard, styles);

