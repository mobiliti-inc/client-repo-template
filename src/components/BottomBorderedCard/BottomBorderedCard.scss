@import '~variables';

.content {
    display: grid;
    justify-content: start;
    align-items: center;
    grid-template-columns: 1fr 1fr;
    border-bottom: 1px solid rgba(0, 0, 0, 0.1);
    height: 100px;
    @media (max-width: $mobile-width) {
        align-items: center;
	}
    & > div {
        display: -webkit-box;
        opacity: 0.9;	
        color: $darker-blue;	
        font-family: $hanken-sans;
        font-size: 19px;	
        font-weight: 300;	
        letter-spacing: -0.42px;	
        line-height: 21px;
        display: -webkit-box;
    }

    &-right {
        justify-self: end;
        text-align: right;
    }
}