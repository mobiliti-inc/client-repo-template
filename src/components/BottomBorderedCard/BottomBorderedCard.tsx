import styles from "./BottomBorderedCard.scss";

type BottomBorderedCardProps = {
	leftText: React.ReactNode | string,
	rightText: string
};

export const BottomBorderedCard: React.SFC<BottomBorderedCardProps> = ({
	leftText,
	rightText
}) => (
		<div styleName="content">
			<div styleName="content">{leftText}</div>
			<div styleName="content-right">{rightText}</div>
		</div>
	);

export default CSSModules(BottomBorderedCard, styles);
