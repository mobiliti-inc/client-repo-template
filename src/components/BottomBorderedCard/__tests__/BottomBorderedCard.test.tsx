import BottomBorderedCard from '../BottomBorderedCard';

describe('BottomBorderedCard test', () => {
	const defaultProps = {
		leftText: 'left text',
		rightText: 'right text'
	};
	it('renders', () => {
		const tree = shallow(<BottomBorderedCard {...defaultProps} />);
		matchSnapshot(tree);
	});
});
