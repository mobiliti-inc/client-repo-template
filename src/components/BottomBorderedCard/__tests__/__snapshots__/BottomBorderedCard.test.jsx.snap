// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BottomBorderedCard test renders 1`] = `
<div
  className="content"
>
  <div
    className="content"
  >
    left text
  </div>
  <div
    className="content-right"
  >
    right text
  </div>
</div>
`;
